src/coreclr/inc/corinfo.h

```diff
@@ -3083,8 +3160,8 @@ class ICorStaticInfo
     // Classifies a swift structure into primitives or an implicit byref for ABI purposes.
     virtual void getSwiftLowering(CORINFO_CLASS_HANDLE structHnd, CORINFO_SWIFT_LOWERING* pLowering) = 0;
 
-    virtual uint32_t getLoongArch64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cls) = 0;
-    virtual uint32_t getRISCV64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cls) = 0;
+    virtual FpStructInRegistersInfo getLoongArch64PassFpStructInRegistersInfo(CORINFO_CLASS_HANDLE cls) = 0;
+    virtual FpStructInRegistersInfo getRiscV64PassFpStructInRegistersInfo(CORINFO_CLASS_HANDLE cls) = 0;
```

Comment:
```suggestion
    virtual FpStructInRegistersInfo getLoongArch64PassFpStructInRegistersInfo(CORINFO_CLASS_HANDLE cls) = 0;
    virtual FpStructInRegistersInfo getRiscV64PassFpStructInRegistersInfo(CORINFO_CLASS_HANDLE cls) = 0;
```
```suggestion
    virtual FpStructInRegistersInfo getFpStructInRegistersInfo(CORINFO_CLASS_HANDLE cls) = 0;
```
Given that `FpStructInRegistersInfo ` struct is not arch specific, should this be a single API? (The actual implementation can still be arch specific.)

Reply:
I'm ok with unifying the API. @shushanhf @LuckyXu-HF ?

Reply:
If we unify these we design things so that RISCV/LA64 altjits won't work properly. Alt jits are not supposed to be full fidelity, but it would still be nice for JIT devs to have the possibility of using them for some quick ad hoc investigations should we need it.

Reply:
I don't mind either way. If unifying the API causes issues elsewhere, I'll leave as is.

Reply:
The implementation of these methods is not factored to support aljits, so the deduping the methods won't make the altjit work any better or worse compared to how they work today.

If we wanted to make altjit fidelity better in cases like this one, I think a better design would be to teach EE side of the JIT/EE interface about the architecture that it is expected to simulate.

Reply:
> The implementation of these methods is not factored to support aljits, so deduping the methods won't make the altjit work any better or worse compared to how they work today.

Ah ok, I had not noticed that.

> If we wanted to make altjit fidelity better in cases like this one, I think a better design would be to teach EE side of the JIT/EE interface about the architecture that it is expected to simulate.

Sounds reasonable.